crates/abq_cli/src/args.rs

```diff
@@ -318,6 +318,18 @@ pub enum Command {
         #[clap(long, default_value = "7")]
         batch_size: NonZeroU64,
 
+        // 🤔 right now this feels like more of a flag than a config option. Should we just make it a flag? `--group-by-file`?
+        // I can imagine expanding on this in the future, but maybe then we can make it an option.
+        /// How we will traverse the tests
+        ///
+        /// By default, ABQ will distribute the next unassigned test to a worker asking for tests.
+        /// work-strategy=by-file, abq will distribute whole-files-at-a-time to workers asking for tests.
+        /// The non-default behavior can be helpful if you have very expensive setup or teardowns in a test file
+        /// that would, by default, be run multiples times by multiples workers instead of a single time on the worker
+        /// responsible for that test file.
+        #[clap(long, default_value = "default")]
+        work_strategy: WorkStrategy,
```

Comment:
Kind of a nit, but maybe "test_strategy"? Since the interface to ABQ users is all about tests.